pkg/util/ssh/os.go

```diff
@@ -221,12 +221,12 @@ func FirewallEnabled(s Interface) (enabled bool, err error) {
 }
 
 func SelinuxEnabled(s Interface) (enabled bool, err error) {
-	stdout, _, exit, err := s.Exec("selinuxenabled")
-	if err != nil || exit != 0 {
+	// https://www.thegeekdiary.com/how-to-check-whether-selinux-is-enabled-or-disabled/
+	stdout, _, _, err := s.Exec("selinuxenabled;  echo $?;")
```

Comment:
I think you can use the the third return value for Exec function rather than run "echo $?" manually